File: linguagens/Python/Exercicios/soma_impares.py

x1 = int(input("Digite dois numeros: "))
x2 = int(input(""))
soma = 0 

if x1 > x2:  
    troca  = x1
    x1 = x2 
    x2 = troca


for i in range (x1 + 1 , x2):
     
    if i % 2 == 1:
        soma = soma + i;   



print (f"Soma dos impares: {soma}")

File: linguagens/C++/exercicios/soma_impares.cpp
#include <iostream> 
#include <cmath> 

using namespace std; 

int main()
{
   int x; 
   int y; 
   int soma = 0;
   int troca = 0; 
   
   cout << "Digite dois numeros: ";
   cin >> x >> y; 

   if( x > y)
   {
        troca = x; 
        x = y; 
        y = troca; 
   } 

   for (int i = x + 1; i < y; i++)
   {
        if(i %2 != 0)
        {
            soma = soma + i; 
        }
   }

   cout << "Soma dos impares: " << soma << endl;

    return 0; 
}